api/v1/controllers/collectors.js

```diff
@@ -206,6 +206,37 @@ function deregisterCollector(req, res, next) {
 } // deregisterCollector
 
 /**
+ * Reregister a collector. Access restricted to Refocus Collector only.
+ *
+ * @param {IncomingMessage} req - The request object
+ * @param {ServerResponse} res - The response object
+ * @param {Function} next - The next middleware function in the stack
+ */
+function reRegisterCollector(req, res, next) {
+
+  //TODO: remove token verification once it's set up in middleware for collectors
```

Comment:
The middleware has already been setup. Check for req.headers.IsCollector to make sure that it is a collector.